assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -559,6 +561,54 @@ export const DEFAULT_NOTIFICATIONS = {
 			);
 		},
 	},
+	'web-data-stream-not-available-notification': {
+		Component: WebDataStreamNotAvailableNotification,
+		priority: 10,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_ABOVE_NAV,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		isDismissible: false,
+		checkRequirements: async ( { select, resolveSelect }, viewContext ) => {
+			const viewOnly =
+				SITE_KIT_VIEW_ONLY_CONTEXTS.includes( viewContext );
```

Comment:
We won't need to check this now as the valid `viewContext`s defined above only allow this on the `VIEW_CONTEXT_MAIN_DASHBOARD` page. So the view context here will always just be `VIEW_CONTEXT_MAIN_DASHBOARD` based on the filtering that happens in the datastore before we execute `checkRequirements`.